src/@next/IndexTable/IndexTable.tsx

```diff
@@ -37,8 +46,19 @@ const IndexTable = ({
       <PolarisIndexTable
         bulkActions={bulkActions}
         checkbox={renderCheckboxHeader}
-        itemCount={itemCount}
+        itemCount={loading ? 0 : itemCount}
         selectedItemsCount={selectedItemsCount}
+        emptySearchTitle={emptySearchTitle ?? ''}
+        emptyState={
```

Comment:
The strategy is to replace the emptyState when the loading prop is `true`. To make it works, I should set the itemCount = `0` too.